<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1 (Python 3.12.4 on win32)" generated="2025-03-06T10:48:43.563015" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\Sruthi_kfc\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-03-06T10:48:43.609252" elapsed="2.502242"/>
</kw>
<status status="PASS" start="2025-03-06T10:48:43.608048" elapsed="2.503446"/>
</test>
<status status="PASS" start="2025-03-06T10:48:43.565532" elapsed="2.548006"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-03-06T10:48:43.555489" level="WARN">Error in file 'c:\Users\Sruthi_kfc\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
